terraform/config.tf

```diff
@@ -1,5 +1,10 @@
 provider "aws" {
   region = "us-east-1"
+  version = "~> 1.60.0"
```

Comment:
This upgrade causes the following changes:
```
  ~ module.container_service_cluster.aws_iam_role.container_instance_ec2
      force_detach_policies:                     "" => "false"

  ~ module.container_service_cluster.aws_iam_role.ecs_service_role
      force_detach_policies:                     "" => "false"
```

These default values were introduced in [0.1.0](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#010-june-20-2017) of the AWS provider.

Reply:
Gotcha. Keeping the provider at 1.60.0 makes sense here.